client/my-sites/customer-home/main.jsx

```diff
@@ -150,6 +166,19 @@ class Home extends Component {
 						description={ translate( "You've completed each item in your checklist." ) }
 					/>
 				) }
+				{ hasChecklistData && isChecklistComplete && siteIsUnlaunched && (
+					<Banner
+						dismissPreferenceName="customer-home-unlaunched-reminder"
+						dismissTemporary={ true }
+						icon="info"
+						callToAction={ translate( 'Launch site' ) }
+						title={ translate( 'Your site is not launched' ) }
+						description={ translate(
+							"If you don't launch your site, it will only be accessbile to specified site members."
```

Comment:
This is new copy.

Reply:
Hi @jblz, thanks for the ping! 

I think the checklist item looks great. I have a suggestion for the nudge copy to reframe it in tone that's a tiny bit more positive. Let me know what you think!

Original: 

> Your site is not launched
If you don't launch your site, it will only be accessible to specified site members.

Proposed: 
> Your site is private
Only you and those you invite can view your site. Launch your site to make it visible to the public.

Reply:
> Your site is private

Thanks @kristastevens. I'll defer to you on the copy, so ignore my original comment @jblz :)